CONTRIBUTING.md

```diff
@@ -2,24 +2,37 @@
 
 ## Getting Started
 
+1. Install Go and Nodejs
 1. Fork the repository on GitHub
 2. Clone the forked repository to your local machine
-3. Build the project: `go build ./...`
-4. Run the project: `./gocert`
+3. Build the frontend: `cd ui && npm build build`
+3. Install the project: `go install ./...`
+4. Run the project: `gocert --config config.yaml`
```

Comment:
This `config.yaml` does not exist? Should we include an example one in the project?

Reply:
No, we could either add another step to create a cert.pem, key.pem and the config.yaml, or include example files of each if we want this to work out of the box.

Reply:
I believe ncluding an example `config.yaml` would be useful. The `.pem` files are not as useful though, those are quite standard files.

Reply:
what about just putting the contents of the config.yaml in this file? I don't want to add files to the top level folder if i can avoid it.

Reply:
this is how it looks: https://github.com/canonical/gocert/blob/TLSENG-226-frontend-skeleton/CONTRIBUTING.md

Reply:
If we want to embed it in another file, I'd suggest using the README.md as it will not only be useful to contributors but to any other users as well.

Reply:
moved it there